formats/shared/definitions/_specialist_document.jsonnet

```diff
@@ -1204,7 +1204,7 @@
           "park-homes---entitlement-to-terminate-an-agreement-for-occupation",
           "rents---market-rent-assured-shorthold-tenancy",
           "rents---fair-rent",
-          "right-to-buy---rtb-appeals-elderly-persons",
+          "right-to-buy---right-to-buy---right-to-buy-appeals-elderly-persons",
```

Comment:
this should be: `right-to-buy---right-to-buy-appeals-elderly-persons` (according to: https://github.com/alphagov/specialist-publisher/blob/master/lib/documents/schemas/residential_property_tribunal_decisions.json#L236)

Reply:
darn, thanks!